curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+The main purpose of the float property in CSS is:
 
 #### --distractors--
 
-Placeholder distractor 1
+To create rounded corners
 
 ---
 
-Placeholder distractor 2
+opacity
 
 ---
 
-Placeholder distractor 3
+To center vertically
 
 #### --answer--
 
-Placeholder answer
+The property is used to align elements to either the left or the right of their container.
```

Comment:
this is the only choice that is a complete answer, distractors and answer should be written in the same stile